lib/ecto/query.ex

```diff
@@ -475,11 +475,34 @@ defmodule Ecto.Query do
   @doc """
   Resets a previously set field on a query.
 
-  It can reset any query field except the query source (`from`).
+  It can reset many fields except the query source (`from`).
 
   ## Example
 
-      query |> Ecto.Query.exclude(:select)
+      Ecto.Query.exclude(query, :join)
```

Comment:
I should write a note that says that this will remove _all_ joins and not just the default inner_join.

Reply:
Yes, please!

Reply:
added